packages/datagateway-common/src/state/reducers/createReducer.tsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Both `table` and `search` can probably just import this function from `common` rather than having their own identical version